ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/activity/PaywallActivityLauncher.kt

```diff
@@ -18,6 +18,14 @@ import com.revenuecat.purchases.ui.revenuecatui.helpers.shouldDisplayPaywall
 @ExperimentalPreviewRevenueCatUIPurchasesAPI
 interface PaywallResultHandler : ActivityResultCallback<PaywallResult>
 
+/**
+ * Implement this interface to receive whether the paywall was displayed when it depends on a condition.
+ */
+@ExperimentalPreviewRevenueCatUIPurchasesAPI
+interface PaywallDisplayCallback {
+    fun onPaywallShouldDisplay(shouldDisplayPaywall: Boolean)
```

Comment:
This sounds very confusing. The parameter is named as a question instead of a noun, as if it was a function. How about:
```suggestion
    fun onPaywallDisplay(wasDisplayed: Boolean)
```

Reply:
Yeah, I was unsure naming this... `onPaywallDisplay` is not fully correct, since it's possible the paywall was not displayed so I had some back and forth in my mind on this... What do you think of `fun onPaywallDisplayResult(wasDisplayed: Boolean)`?

Reply:
I did this rename in https://github.com/RevenueCat/purchases-android/pull/1542/commits/2fad81932bee82b4e94091d5d65e2ac252ff52e1. Lmk what you think!